libuavcan/test/node/test_node.hpp

```diff
@@ -74,6 +75,15 @@ struct PairableCanDriver : public uavcan::ICanDriver, public uavcan::ICanIface
         return NULL;
     }
 
+    virtual const uavcan::ICanIface* getIface(uavcan::uint8_t iface_index) const
+    {
+        if (iface_index == 0)
+        {
+            return this;
+        }
+        return NULL;
+    }
```

Comment:
This overload is no longer needed, because there's a default virtual implementation in the base class that does the same thing ([line 180](https://github.com/UAVCAN/libuavcan/pull/41/files#diff-dfb72932e25cf3185462fb93974efa06R180)).